test/facade_converter_test.ts

```diff
@@ -54,7 +69,7 @@ function getSources(str: string): {[k: string]: string} {
           static get(m: any, k: string): number { return m[k]; }
         }
         var global: any;
-        export var Promise = global.Promise;`,
+        export var Promise = 1;`,
```

Comment:
Still don't get it :-)

So the problem is that we find a global variable called Promise, but before it was an alias, and we resolved through the alias? Maybe change it to something like `{definedHere: true}` and add a comment?